server/src/controllers/MarkerController.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from 'express';
+import Marker from '../models/Marker';
+
+// GET endpoint to retrieve all markers with longitude and latitude
+export const getMarkers = async (req: Request, res: Response) => {
+    try {
+        // Retrieve all markers from the database with projection to include only longitude and latitude
+        const markers = await Marker.find({}, 'longitude latitude');
+        return res.status(200).json(markers); // Return retrieved markers as JSON response
+    } catch (err) {
+        console.error('Could not get markers: ', err);
+        return res.status(500).json({ message: 'Internal server error' }); // Handle errors gracefully
+    }
+};
+
+// POST endpoint to save a marker to the database
+export const saveMarker = async (req: Request, res: Response) => {
+    const { longitude, latitude } = req.body;
+
+    // Check if longitude and latitude are provided
+    if (!longitude || !latitude) {
+        return res.status(400).json({ message: 'Longitude and Latitude are required' });
+    }
```

Comment:
This is a good example of what I recommend doing. This is nicely documented and clear to whoever is using the API.